### PR TITLE
[EN-284]: Not all queries in commercetools are wrapped in `gql` template literal

### PR DESCRIPTION
**Pull request by lsliwaradioluz**
### Related Issues

Closes [EN-284](https://vsf.atlassian.net/browse/EN-284)

### Short Description of the PR

Found and wrapped in gql all queries where the wrapper had been missing

### Pull Request Checklist

- [ ] I have updated the Changelog ([V1](https://github.com/DivanteLtd/vue-storefront/blob/develop/CHANGELOG.md)) [v2](https://docs-next.vuestorefront.io/contributing/creating-changelog.html) and mentioned all breaking changes in the public API.
- [ ] I have documented all new public APIs and made changes to existing docs mentioning the parts I've changed so they're up to date.
- [ ] I have tested my Pull Request on production build and (to my knowledge) it works without any issues

- I tested manually my code and it works well with both:
- [ ] Default Theme
- [ ] Capybara Theme
- [ ] I have written test cases for my code

- [ ] I have followed [naming conventions](https://github.com/kettanaito/naming-cheatsheet)

### CODE REVIEW

**Review by filipsobol — APPROVED** [review]

File: packages/commercetools/api-client/__tests__/api/customerUpdateMe.spec.ts [hunk 1]
```diff
@@ -1,6 +1,6 @@
 import customerUpdateMe from '../../src/api/customerUpdateMe';
 
-jest.mock('../../src/api/customerUpdateMe/defaultMutation.ts', () => '');
+jest.unmock('../../src/api/customerUpdateMe/defaultMutation.ts');
```
Comment (Baroshem) [inline packages/commercetools/api-client/__tests__/api/customerUpdateMe.spec.ts:3]: Can you explain why there is an unmock?

**Review by Baroshem — COMMENTED** [review]
One question

File: packages/commercetools/api-client/__tests__/api/customerUpdateMe.spec.ts (hunk 1 above)
Comment (lsliwaradioluz) [reply packages/commercetools/api-client/__tests__/api/customerUpdateMe.spec.ts]: @Baroshem I think it's redundant. Deleted.

**Review by alefbarbeli — APPROVED** [review]

**Comment by lukaszjedrasik** [comment]
@lsliwaradioluz could you resolve conflicts, please?

**Review by lukaszjedrasik — APPROVED** [review]

**Comment by lukaszjedrasik** [comment]
@lsliwaradioluz it's okay from my side. I think you can merge it.

**Pull request merged**